src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
+@import "./utils/resets";
+@import "./utils/variables";
+@import "./utils/loop";
+@import "./blocks/calendar.scss";
+
+.main__calendar--position--center {
```

Comment:
no need to create a bem modifier for centering

Reply:
you can use mix like here
<img width="954" alt="image" src="https://user-images.githubusercontent.com/102411130/219595280-58f893ac-b2fd-478b-a1bb-9916f17bdc32.png">